### ISSUE TITLE
producer_args is deprecated

### ISSUE THREAD

**Issue by KES777** [frioux/DBIx-Class-DeploymentHandler#77]
```
SQL::Translator::Diff::schema_diff(): producer_args is deprecated -- it does not go straight to the producer, it goes to the internal sqlt object. Please use sqlt_args, which reflects how it's used at DBIx/Class/DeploymentHandler/DeployMethod/SQL/Translator.pm line 533
```

https://github.com/dbsrgits/sql-translator/blob/master/Changes#L4

Just a small renaming.

**Comment by rabbiveesh** [comment]
Yeah, i meant to open this ticket before I released, sorry about that

**Comment by wesQ3** [comment]
Fixed in v0.002234
